docs/tutorial.rst

```diff
@@ -3,6 +3,113 @@
 ========
 Tutorial
 ========
+There are two main ways of accessing the resources of the ``stdpopsim`` package
+that will be detailed in this tutorial. The first is via the command line
+interface (CLI). This is useful if you want to do a straightforward run of the
+models in the ``stdpopsim`` :ref:`catalog <sec_catalog>`. The other way to
+access the ``stdpopsim`` resources is via the python API. This is a bit more
+complicated however it allows for more advanced tasks. This tutorial will walk
+through both ways of using the ``stdpopsim`` package as well as some more
+advanced tasks you may wish to do.
+
+***************************
```

Comment:
As per previous comment, it would be good to put `.. sec_cli_tute:` above this heading so that we can link to it later on. Ariella and I will be adding some more tutorials to this page soon, so this CLI tute may end up further down on the page, and the `.. sec_tutorial:` command is for the whole page, not specific to this particular CLI tutorial.